spec/lightspeed/client_spec.rb

```diff
@@ -33,18 +33,18 @@
       accounts = client.accounts
       expect(accounts).to be_a(Lightspeed::Accounts)
       expect(accounts.length).to eq(1)
-      expect(accounts.first.id).to be(Lightspeed::TEST_ACCOUNT_ID)
+      expect(accounts.first.id).to be(117102)
```

Comment:
I'd love to do better, not sure how. The response is going to depend on the account ID provided, so the `request_matcher` trick doesn't help here.